src/client/components/ProductsComponent/ProductsComponent.stories.js

```diff
@@ -0,0 +1,314 @@
+import React from 'react';
+
+import ProductsComponent from './ProductsComponent';
+
+export default {
+  title: 'Components / Products Component',
+  component: ProductsComponent,
+  argTypes: {
+    products: { control: '' },
+  },
+};
+
+const template = ({ products }) => {
+  return (
```

Comment:
Is there a reason why you choose not to showcase the sorting here also?

Reply:
Got error i think .. with update of sorted products.. 
Thats why I did like this..

Reply:
Sorry, my mistake. The sorting is working here.